README.md

```diff
@@ -1,4 +1,50 @@
-# o-big-number [![Build Status](https://circleci.com/gh/Financial-Times/o-big-number.png?style=shield&circle-token=d4cdedae5153e5aa0318dcb53236ac885808a66a)](https://circleci.com/gh/Financial-Times/o-big-number)
-Big Number component [![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](#licence)
+# o-big-number [![Build Status](https://circleci.com/gh/Financial-Times/o-big-number.png?style=shield&circle-token=9ca314332de2a9b6a80eb8477e097d9acbc96e0b)](https://circleci.com/gh/Financial-Times/o-big-number) [![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](#licence)
 
-Coming soon.
+Typographical styles to highlight and describe a big number. Positing of the big number, for example to the left in a style of a pull-quote, is left to the user. This is so o-big-number may be used in different contexts without overrides.
```

Comment:
```suggestion
Typographical styles to highlight and describe a big number. Positioning of the big number, for example to the left in a style of a pull-quote, is left to the user. This is so o-big-number may be used in different contexts without overrides.
```

Reply:
Do our customers know what we mean by "overrides" here? Does it mean writing extra CSS to remove the positioning that would have been inside the o-big-number CSS?

Reply:
Yeah that's right. Good point, I'll make that more clear!